refs/schemas/priv/RFC-PRIV.md

```diff
@@ -0,0 +1,706 @@
+
+### Design Choices
+
+We have made the following choices:
+- **Language Independence**. The Privacy Request Interchange Vocabulary is independent from any programming language, or any format or language for expressing structured data, and can be materialised in different forms such as json, xml, or other. A [json schema](PRIV.schema.json) is provided for convenience.
+
+- **Rich Semantics**. The Privacy Request Interchange Vocabulary includes `terms` - reserved words to describe common types of Privacy Requests, categories of data, categories of data processing and other key notions. This choice is made to facilitate their uniform interpretation by the implementing systems. Their [human-readable titles and descriptions](dictionary) are provided in json format for convenience.
+
+- **Multiple User Identities**. The Privacy Request Interchange Vocabulary  allows for a Data Subject to be identified using more than one user identity. This choice is made to enable the Privacy Request to be easily exchanged across Systems that use different user identifiers.
+
+- **Systems resolve Privacy Requests**. The Privacy Requests are interpreted at the level of a particular System. If an Organisation operates several Systems, and if the Data Subject wants to have the Privacy Request transmitted to all of them, each System may respond differently. While a Privacy Request can target a group of Systems, the most atomic target of a Privacy Request is thus the System exposing an API for Privacy Requests, and it is only at this level that a Privacy Request can be resolved.
+
+- **Decentralised IDs**. The Privacy Request Interchange Vocabulary uses decentralised ways to uniquely identify Data Subjects, Systems, Requests and their elements. The exchange of Privacy Requests can happen without a centralised entity to control identity disambiguation.
+
+## Proposal
+
+The Privacy Request Interchange Vocabulary includes the following:
+- Key Concepts: [Consent](#consent), [Data Capture](#data-capture), [Data Capture Fragment](#data-capture-fragments), [Demand](#demands), [Privacy Request](#privacy-request), [Privacy Scope](#privacy-scope)
+- Properties: **TBD** full list of properties
+- Terms: **TBD** full list of terms
+
+### Privacy Request
+
+Data Subject is the author of a Privacy Request.
+
+| Property | Expected cardinality | Expected values |
+| --------------- | ------ | -------------------- |
+| `data-subject` |  1-* | [Data Subject Identities](#decentralized-identity-of-data-subjects) each containing one `dsid` and one `dsid-schema`|
+
+A System MAY have multiple ways to identify the Data Subject, especially when data about them came from some other System that uses different identifiers.
+
+The System capturing the Privacy Request MAY associate multiple Data Subject Identities to the Privacy Request, especially if the Privacy Request is likely to be transmitted to other systems.
```

Comment:
> Do we care about the author of the Privacy Request?

I think we must asume that the author of the Privacy Request = Data Subject, and that they MUST be able to authenticate as such, otherwise they can't be given any answer as data might be confidential.

>  So, is it correct to interpret that one Privacy Request may have multiple Data Subject Identities, but all identities are related to one Data Subject?
Yes. A Privacy Request relates to one and only one Data Subject.

Reply:
I'm including some further clarification in PRIV to clarify this. Thanx for pointing it out.